src/reliability.jl

```diff
@@ -51,7 +51,13 @@ item 4: 0.7826
 ```
 """
 function cronbachalpha(covmatrix::AbstractMatrix{<:Real})
-    isposdef(covmatrix) || throw(ArgumentError("Covariance matrix must be positive definite."))
+    if !isposdef(covmatrix)
+        msg = """
+            Covariance matrix must be positive definite.
+            Did you use `cronbachalpha(cov(Matrix(...)))`?
+            """
+        throw(ArgumentError(msg))
```

Comment:
Thanks. How about something like this? Calling `Matrix` shouldn't be needed as the input is already a matrix.
```suggestion
         throw(ArgumentError("Covariance matrix must be positive definite. " *
                             "Maybe you passed the data matrix instead of its covariance matrix? " *
                             "If so, call `cronbachalpha(cov(...))` instead.")
```

Reply:
Thanks. Makes sense